### ISSUE TITLE
Bug: Moving Items from Containers to New Actor Breaks Sheet

### ISSUE THREAD

**Issue by justinthejaguar** [vttred/ose#144]
Items added to a container cannot be dragged to another actor without breaking the sheet encumbrance and treasure value.
To recreate (to show both value and weight):

1. Create actor1.
2. Create container on actor1
3. Add GP Item to actor1 (make sure there is a value).
4. Move GP to container on actor1
5. Create actor2.
6. Drag GP from actor1 to actor2.

**Result**: GP item will disappear. Weight and treasure value will increase.

**Issue**: Adding items to a container gives the actor ownership of the item. This prevents the second actor from seeing the item on their sheet, but it still raises encumbrance and treasure value.

**Fix:** Remove ownership on drag from container? I am not really certain what fixes are best for this.

![Screen Shot 2022-03-13 at 11 50 45 PM](https://user-images.githubusercontent.com/38022481/158101933-09ecb240-93c0-4f8b-979b-86303b8051eb.png)

**Comment by anthonyronda** [comment]
Ah, I see, this is a variant of behavior I'm trying to fix in #30. This one should be much easier to fix

**Comment by anthonyronda** [comment]
Macro to temporarily fix this for each affected actor:
```js
/* Macro to fix items that may be stuck hidden because
 * they have a containerId value but the container can't be found
 * or two containers that contain each other */

// use with token selected
// updates the actor and linked tokens, not unlinked tokens
// If it's an unlinked token, delete the token afterward and create a new one

let arr = [];
actor.data.items.forEach((item) => {
  if (item.data.type === "container")
    arr.push({ _id: item.id, "data.containerId": "", "data.itemIds": []});
  else
    arr.push({ _id: item.id, "data.containerId": ""});
})
actor.updateEmbeddedDocuments("Item", arr);
```
